src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -78,6 +130,20 @@ describe('ReviewScreen', () => {
         'fiatConnectReviewScreen.paymentMethodVia, {"providerName":"Provider Two"}',
       ])
     })
+    it('shows fee disclaimer when quote currency does not match user locale currency', () => {
+      const mockProps = getProps(CICOFlow.CashIn)
+      //Re-renders, need to mock twice
```

Comment:
nit: might be cleaner if you just set a default mock in before each, and drop the once when mocking here so the dependency on how many times the component renders is removed.

Reply:
Good idea!